README.md

```diff
@@ -0,0 +1,26 @@
+
+# Getting started
+
+Pixi is a multi-container Docker application; there's a NodeJS application
+container and a MongDB database container.
+
+The application is defined in the `docker-compose.yaml` file; to run it,
+make sure you have `docker-compose` installed
+([instructions](https://docs.docker.com/compose/install/)) and then run:
+
+```
+docker-compose up
+```
+
+This will expose the following ports:
```

Comment:
`resources` or `services` might be better than "ports".

The URLs could be proper markdown links.

Reply:
You know what, when I was reviewing it i was just seeing the text and not thinking it through. You're right github will render it as a link and it'll be fine. Disregard....sry. :zap: